Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-projectm/) for Kodi add-ons: look and feel/visualization.projectm.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-look-and-feel/-/visualization-projectm/horizontal-auto.svg)

gade01 commented:
Contans incorrect translations:
https://github.com/xbmc/visualization.projectm/pull/67/files#diff-4693bf30a5d0a7cdc0b0c937269a0d6a0f26ade0d00f6fdce6f6140e4ccb9fdcR95

https://github.com/xbmc/visualization.projectm/pull/67/files#diff-4693bf30a5d0a7cdc0b0c937269a0d6a0f26ade0d00f6fdce6f6140e4ccb9fdcR99

etc.

weblate committed:
Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 86.2% (25 of 29 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 100.0% (29 of 29 strings)

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: 撷星芒 <mofa2020@qq.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-look-and-feel/visualization-projectm/zh_cn/
Translation: Kodi add-ons: look and feel/visualization.projectm

gade01 reviewed (approved):
I removed the wrong translations.
